classic_pagination.gemspec

```diff
@@ -0,0 +1,15 @@
+Gem::Specification.new do |s|
+  s.name        = 'classic_pagination'
+  s.version     = '1.0.1'
```

Comment:
Does this need to match [line 4](#diff-89cade48462044ee1b672dc5f4c3ec250fbd29effcd8932096a23c1283c6731fR4) above?